<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EasyCryptWindow</class>
 <widget class="QMainWindow" name="EasyCryptWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>212</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>212</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>212</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>EasyCrypt</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>407</width>
      <height>196</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="mainHorizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="leftHorizontalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="textLabel">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Text:</string>
         </property>
         <property name="buddy">
          <cstring>textEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QTextEdit" name="textEdit">
           <property name="minimumSize">
            <size>
             <width>256</width>
             <height>190</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>256</width>
             <height>190</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="mainVerticalLayout">
       <item>
        <layout class="QHBoxLayout" name="topRightHorizontalLayout">
         <item>
          <widget class="QLabel" name="numberLabel">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>&amp;Number of times:</string>
           </property>
           <property name="buddy">
            <cstring>numberLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="numberLineEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="bottomRightVerticalLayout">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QPushButton" name="encryptButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>&amp;Encrypt</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="decryptButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>&amp;Decrypt</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="closeButton">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>&amp;Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>textEdit</tabstop>
  <tabstop>numberLineEdit</tabstop>
  <tabstop>encryptButton</tabstop>
  <tabstop>decryptButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
